Cython/Compiler/TypeSlots.py

```diff
@@ -1064,6 +1064,7 @@ def __init__(self, old_binops):
             EmptySlot("tp_finalize", ifdef="PY_VERSION_HEX >= 0x030400a1"),
             EmptySlot("tp_vectorcall", ifdef="PY_VERSION_HEX >= 0x030800b1"),
             EmptySlot("tp_print", ifdef="PY_VERSION_HEX >= 0x030800b4 && PY_VERSION_HEX < 0x03090000"),
+            EmptySlot("tp_inline_values_offset", ifdef="PY_VERSION_HEX >= 0x030B00A2"),
```

Comment:
I've assumed this slot will come before `tp_pypy_flags` although I don't think we _know_ this right now.